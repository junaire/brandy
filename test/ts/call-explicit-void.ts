var x: bigint = 5n;
callPrint(x);

function callPrint(x: bigint): void {
    console.log(x);
}
