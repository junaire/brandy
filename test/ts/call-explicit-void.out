@main {
  v0: int = const 5;
  x: int = id v0;
  v1: int = id x;
  call @callPrint v1;
  v2: int = const 0;
}
@callPrint(x: int) {
  v0: int = id x;
  print v0;
  v1: int = const 0;
}
